events/code_review_events/workflow.py

```diff
@@ -334,20 +506,46 @@ def __init__(self, cache_root):
         # Create pulse listener for unit test failures
         if self.workflow.publish:
             self.pulse = PulseListener(
-                QUEUE_PULSE,
-                "exchange/taskcluster-queue/v1/task-completed",
-                "*.*.gecko-level-3._",
+                QUEUE_PULSE_TRY_TASK_END,
+                [
+                    (
+                        "exchange/taskcluster-queue/v1/task-completed",
+                        ["#.gecko-level-1.#"],
+                    ),
+                    (
+                        "exchange/taskcluster-queue/v1/task-failed",
+                        ["#.gecko-level-1.#"],
+                    ),
+                    (
+                        "exchange/taskcluster-queue/v1/task-exception",
+                        ["#.gecko-level-1.#"],
+                    ),
+                ],
                 taskcluster_config.secrets["pulse_user"],
                 taskcluster_config.secrets["pulse_password"],
             )
             self.pulse.register(self.bus)
 
+            self.community_pulse = PulseListener(
+                QUEUE_PULSE_BUGBUG_TEST_SELECT,
+                [
+                    (
+                        "exchange/taskcluster-queue/v1/task-completed",
+                        ["route.project.relman.bugbug.test_select"],
+                    ),
+                ],
+                taskcluster_config.secrets["pulse_user"],
+                taskcluster_config.secrets["pulse_password"],
+                "/communitytc",
+            )
+            self.community_pulse.register(self.bus)
+
     def run(self):
         consumers = [
             # Code review main workflow
-            self.workflow.run(),
+            # self.workflow.run(),
             # Add mercurial task
-            self.mercurial.run(),
+            # self.mercurial.run(),
```

Comment:
Let's not forget to remove those

Reply:
Why? This was the one change that reduced the risk of new bugs to 0!

Reply:
:drum: